Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM fedora:25 as builder
+
+RUN dnf install -y \
+    git-core \
+    gcc \
+    gcc-c++ \
+    boost-devel \
+    cmake \
+    autoconf \
+    automake \
+    libtool \
+    mariadb-devel \
+    file \
+    hostname \
+    mariadb-server
+
+RUN git clone --branch master --single-branch --depth 1 https://github.com/anton-povarov/meow /_src/meow
+RUN git clone --branch master --single-branch --depth 1 https://github.com/nanomsg/nanomsg /_src/nanomsg
+
+COPY . /_src/pinba2
+RUN /_src/pinba2/docker/build-from-source.sh
+
+FROM fedora:25
+MAINTAINER Anton Povarov "anton.povarov@gmail.com"
+
+RUN dnf install -y file hostname mariadb-server
```

Comment:
@anton-povarov do you really need `file` for production instance? The same question is about `hostname` but IMHO it might be just missing dependencies for `mariadb-server`.